src/components/modals/ManageAntModal/ManageAntModal.tsx

```diff
@@ -268,7 +307,7 @@ function ManageAntModal() {
                               getTransactionPayloadByInteractionType(
                                 CONTRACT_TYPES.ANT,
                                 row.interactionType,
-                                modifiedValue!,
+                                modifiedValue!.toString(),
```

Comment:
remove the force case (`!`) here and update the function to handle a potentially `undefined` for this param

Reply:
added